webapp/src/shared/components/Row/ScoreRow.tsx

```diff
@@ -0,0 +1,104 @@
+import './Row.css';
+import { TextColor, TextVariant, TextWeight } from '../Text/Text';
+import { default as Text } from '../Text/Text';
+import { MediumAvatar } from '../Avatar/Avatar';
+import { Link } from 'react-router-dom';
+import { ScoreProps } from '../Score/Score';
+
+export type ScoreRowProps = {
+  scoreProps: ScoreProps;
+  url?: string;
+  gameMode?: string;
+  gameDuration?: number;
+  date?: Date;
+};
+
+export default function ScoreRow({
```

Comment:
praise: I like how you designed and implemented this component. Looks great!

Reply:
Thanks